src/js/bookmarks.js

```
@@ -0,0 +1,314 @@
+    },
+    getCount() {
+      return bookmarkedSet.size;
+    },
+    clear() {
+      bookmarkedSet.clear();
+      bookmarkOrder = [];
+      saveBookmarks(bookmarkedSet, bookmarkOrder);
+    },
+    getSortedOrgs(sortKey = "added") {
+      const ordered = bookmarkOrder
+        .map((name) => ORGS.find((o) => o.name === name))
```

ESLint

[error] 'ORGS' is not defined.

(no-undef)